e2e/js_run_devserver/multirun_test.sh

```diff
@@ -14,107 +14,119 @@ _sedi () {
   sed "${sedi[@]}" "$@"
 }
 
-echo "TEST - $0: $1"
+echo "$$: TEST - $0: $1"
 
-./node_modules/.bin/ibazel run "$1" "$BZLMOD_FLAG" >/dev/null 2>&1 &
+./node_modules/.bin/ibazel run "$1" "$BZLMOD_FLAG" 2>&1 &
 ibazel_pid="$!"
 
 function _exit {
-  echo "Cleanup..."
+  echo "$$: Cleanup..."
   kill "$ibazel_pid"
+  wait "$ibazel_pid"
```

Comment:
One error was port 8080 not being available, I assume because the previous process was not dead yet and still using the port. This should wait for it to fully die if I'm understanding correctly.